### DISCUSSION TITLE
new widget : pagination for Loop Grid

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#23022] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team!

In version 3.15 you released a standalone widget for filtering categories and tags that can be connected to the desired Loop Grid widget and work. It was a great decision.

But it would be great if you also release a new widget for pagination. We currently have pagination in the loop grid widget, but it's very simple. Developing a new widget is much easier than developing the current layout in the Loop Grid widget.

### Describe the solution you'd like

I have a few suggestions for pagination in Loop Grid. Please check them out. It is definitely the most standard and best way to paginate in Loop Grid.

1- Remove the current pagination section in the Loop Grid widget. **Keep only Infinite Scroll and Load on Click pagination in Grid loop widget**.

2- Create a new widget for the pagination. This widget only contains numeric pagination. Also add an option to include text for (next & prev) if needed. T**his pagination widget should have the required options. Features such as :**

A: Width and height for the pagination box.
B: Custom icons for previous and next arrows
C: End offset
D: Mid offset

![pagination](https://github.com/elementor/elementor/assets/88034902/396b1106-8637-4234-8374-ff5381adfb80)

E: Ajax pagination
F: auto scroll to top After Chenge Page

This will be a wonderful widget. It should be noted that bricks builder takes advantage of the above items (except item F). I hope Elementor will add it too.

**Comment by 79ho3ein** [comment]
nice👌

**Comment by dave-gil** [comment]
+1
